tests/test_Globals.cpp

```diff
@@ -1,11 +1,13 @@
 #include "test_Globals.hpp"
 
-#include <GameConfig.hpp>
+#include <RWConfig.hpp>
 
 #if RW_TEST_WITH_DATA
 std::string Global::getGamePath() {
-    GameConfig config;
-    config.loadFile(GameConfig::getDefaultConfigPath() / "openrw.ini");
-    return config.getGameDataPath().string(); //FIXME: use path
+    rwfs::path configPath = RWConfigParser::getDefaultConfigPath() / "openrw.ini";
+    RWConfigLayer cfgLayer;
+    RWConfigParser cfgParser;
+    std::tie(cfgLayer, std::ignore) = cfgParser.loadFile(configPath);
```

Comment:
I've found that using `[[maybe_unused]]` you can suppress warnings.

Reply:
Indeed. Although Visual Studio 2017 does not (currently) support using it in a structured for loop, as in `for([[maybe_unused} auto& [key, value] : map)`. (I've got a compiler error because of this)

Reply:
I'll change this and test whether the loading was successful.